### ISSUE TITLE
Publish homie entities

### ISSUE THREAD

**Issue by legovaer** [elbowz/ha-homie#4]
Maybe I'm missing something, but at this point home assistant is only subscribing to homie. It would be nice if the home assistant entities also get published.

**Comment by elbowz** [comment]
What you mean with "published"?

Do you have new devices and entities discovered in the `/config/integrations`?

Something like:
![Screenshot_20211220_170316](https://user-images.githubusercontent.com/1451484/146796786-af93c46e-fdec-41e1-bf77-931d4829ca78.png)

**Comment by legovaer** [comment]
I mean like in the last screens hot of #1. There you see another application or service publishing the device data to mqtt. 

It would be nice if home assistant could do the same. There is already some functionality that does exactly that, without the homie convention.

**Comment by elbowz** [comment]
Do you mean a bridge HomeAssistant2Homie?

In the img, you speak about, is always a device that publish information through the homie convention on mqtt.

HA could publish a lot information about registered devices/entities gathered (I guess), it's essentially a hub, but I missing the use case/reason.

**Comment by legovaer** [comment]
Well, it would be nice if home assistant could exactly send that same information. A use case:

I have a [Homey device](https://homey.app/en-ca/homey-pro) that interacts with my RF/IR devices. It's also my dashboard as I'm too lazy to start creating dashboards in lovelace. 

This homey device uses a plugin called [_MQTT HUB_](https://homey.app/en-ie/app/nl.hdg.mqtt/MQTT-Hub/) Basically it publishes the info about the devices on MQTT using the homie convention. 

This allows home assistant to interact with my curtains and other IR/RF devices.

When my girlfriend pushes a button in the UI of Homey, home assistant will update the state of the device because of your integration. 

Now it would be cool that the UI in my Homey device is able to show entities that are only available in Home Assistant. 

![This](https://user-images.githubusercontent.com/5813212/146842532-fea9012e-52b2-47f3-ac0e-6f8ee98058fe.jpeg) is a part of my Homey Dashboard. It's auto-generated without fussing around with tons of cards and other lovelace magic.

So basically:

Home Assistant -----_MQTT HOMIE_-----Homey 

But bi-directional 🤔

**Comment by elbowz** [comment]
So, little recap: 

Hom**ey** publish devices on a MQTT broker, through MQTT HUB plugin, following the Hom**ie** convention?

Using my ha-homie intgretion you have the Homey devices autodiscovered on HA, right?

Would you like the devices present ONLY in HA, published in Homey?

* Are you sure that MQTT HUB not only write devices on MQTT, but also read them if you add an homie device?

Answering myself reading the plugin description:
> - Auto discovery of external devices using the Homie convention.

* Basically you want a bridge HA2MQTT-Homie.

I don't think is a good idea take running all HA environment and use another dashboard (Homey). 

Maybe, It's better write plugin for Homey that support other devices protocol or use only HA...or convince your girlfriend to use HA ;)

I never thought to this use case...but it's really different problem from what my integration do, I think a lot of work behind
